router/jsonrpc_test.go

```diff
@@ -98,9 +104,10 @@ func TestJsonRpcAnonToken(t *testing.T) {
 	}
 	defer os.RemoveAll(dir)
 
-	_, pv, _ := startBVC(t, cfg, dir)
+	_, pv, node := startBVC(t, cfg, dir)
+	defer node.Stop()
 
-	txBouncer := relay.NewWithNetworks(*testnet)
+	txBouncer, err := relay.NewWithNetworks(networks.IndexOf("Badlands"))
```

Comment:
I made the original change, replacing a hard-coded number with `*testnet`, when I should not have.